### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by TadesseDev**
# changes introduced. 😎
---
- Create a method called current_user in my ApplicationController ✔
- Creates a Post on behalf of the `current_user` ✔
- Creates a Comment on behalf of the `current_user` ✔
- Allow Users to add likes to Posts
> # Note
 - > this pull request focuses only on the functionality of the system.` styling and error validation` is yet to come.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -10,7 +10,23 @@ def show
     @post = Post.where(id: params[:id]).first
   end
 
+  def new
+    puts params
```
Comment (whiz25) [inline app/controllers/posts_controller.rb]: - It's not recommended to print values with production code for security reasons. Please remove those puts statements from your code.

**Review by whiz25 — CHANGES_REQUESTED** [review]
Hi @Tadesse-Alemayehu, 

Good job so far!
- All requirements have been fulfilled as expected
- Code is well organized and clean
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TadesseDev** [comment]
Thank you @whiz25, I'll make sure to remove `logs` from my project before the next review request.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Tadesse-Alemayehu👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TadesseDev** [comment]
Thanks, @thecodechaser, Nice to have your review on this.

**Pull request merged**